brianlong/www.validators.app
# 184544114 - add trent-mode page

#### What's this PR do?
PR adds new trent-mode page.

#### How should this be manually tested?
- Enter the `/trent-mode` page. Verify its content. Use search bar and confirm that it works. Switch networks to confirm that they're correctly passed to search query.
- check the `/validators` page and verify that there is no negative impact.

#### What are the relevant tickets?
[- URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/184544114)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
